catalog/ui/search-ui/standard/pom.xml

```diff
@@ -274,6 +274,18 @@
             </activation>
```

Comment:
I don't think you meant to have these doc changes in here. It looks like you're rolling back a patch that was applied yesterday.